app/src/main/java/com/frankenstein/screenx/database/DatabaseManager.java

```diff
@@ -11,7 +12,7 @@
 
     public static ScreenShotDatabase getInstance(Context context) {
         if (_mInstance == null) {
-            _mInstance = Room.databaseBuilder(context, ScreenShotDatabase.class, DB_NAME).build();
+            _mInstance = Room.databaseBuilder(context, ScreenShotDatabase.class, DB_NAME).fallbackToDestructiveMigration().build();
```

Comment:
Add a please do not carry forward here, As destroying data in production system is not ideal